bot/exts/utils/bot_info.py

```diff
@@ -45,7 +45,10 @@ async def stats(self, inter: ApplicationCommandInteraction) -> None:
         embed.add_field(name="Servers", value=len(guilds))
         embed.add_field(
             name="Members",
-            value=sum([guild.member_count for guild in guilds]),
+            value=len(
+                set().union(set(user.id for user in guild.members) for guild in guilds)
+            )  # Length of the union of every single UID
+            - 1,  # Remove the bot itself
```

Comment:
```suggestion
```
The bot is in fact a member, so does it really make sense for us to exclude it? At the end of the day, we're counting the other bots in the guilds.